src/Filter/FilterComposite.php

```diff
@@ -98,8 +104,7 @@ public function addFilter($name, $filter, $condition = self::CONDITION_OR)
     {
         if (!is_callable($filter) && !($filter instanceof FilterInterface)) {
             throw new InvalidArgumentException(
-                'The value of ' . $name . ' should be either a callable or ' .
-                'an instance of Zend\Stdlib\Hydrator\Filter\FilterInterface'
```

Comment:
Where is it placed now?  Shall we just use FilterInterface::class to get the name?